bundles/org.openhab.binding.hydrawise/src/main/java/org/openhab/binding/hydrawise/internal/handler/HydrawiseAccountHandler.java

```diff
@@ -232,8 +255,10 @@ private void poll(boolean retry) {
                 updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, e.getMessage());
             }
         } catch (HydrawiseAuthenticationException e) {
-            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, e.getMessage());
+            logger.debug("Token has been rejected, will try to refresh token in {} secs : {}", TOKEN_REFRESH_SECONDS,
```

Comment:
Sorry, two more questions:
- It it possible to know whether we just need to refresh token, or if this is actually an authentication error that could be caused by configuration?
- Do you think we should still set status OFFLINE, but with COMMUNICATION_ERROR and perhaps a description for this situation?

Reply:
Since you are not available now and the 4.1.1 build is scheduled very soon, I will merge the PR as is and leave this comment open so we can get back to this. In case it leads to any further changes, this can be included in next patch release.

Reply:
Reading the code again, the first question is not relevant since it's a one-time retry, and it will thus still fail as usually after one attempt.